### PR TITLE
(FACT-1757) Update readme and changelog

### PR DESCRIPTION
**Pull request by caseywilliams**
Adds a real readme and fills in the changelog for work done so far. Also
updates the reported version to 0.1.3 (the next release).

We should wait to merge this until #24 and #25 are merged, since it describes those changes.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.2.2)
-project(whereami VERSION 0.1.0)
```
Comment (branan) [inline CMakeLists.txt:2]: Since this is targetting master, should we instead mark the version as 0.2.0?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,115 @@
...
+- Zone
+  - brand : string
+  - id : int
+  - ip_type : string
+  - name : string
+  - uuid : string
+
+## Example usage
+
+```cpp
```
Comment (branan) [inline README.md:63]: Doxygen appears to be falling over trying to do something with this block :(

Comment (caseywilliams) [reply README.md:63]: It seems like we could use the Doxyfile to disable checking for this (it actually explicitly includes README.md), but the Doxyfile is included in .gitignore... where could I change how the Doxyfile is generated?

Comment (branan) [reply README.md:63]: `Doxyfile.in` seems like the best place - we just do some simple variable substitution on that file, AFAICT

Comment (caseywilliams) [reply README.md:63]: D'oh, of course - updated, we'll see how this goes

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,28 +1,37 @@
 # Change Log
-All notable changes to this project will be documented in this file.
-This project adheres to [Semantic Versioning](http://semver.org/).
 
-## [0.2.0] - TBD
+## [0.1.3] - TBD
```
Comment (branan) [inline CHANGELOG.md]: 0.2.0 again!

**Pull request merged**